apps/schematic/api/schematic_api/controllers/schema_controller_impl.py

```diff
@@ -10,9 +10,73 @@
 from schematic_api.models.validation_rule import ValidationRule
 from schematic_api.models.nodes_page import NodesPage
 from schematic_api.models.node import Node
+from schematic_api.models.connected_nodes_page import ConnectedNodesPage
+from schematic_api.models.connected_nodes import ConnectedNodes
 from schematic_api.controllers.utils import handle_exceptions
 
 
+def get_connected_nodes_from_schematic(
+    relationship_type: str,
+    schema_url: str,
+) -> list[list[Any]]:
+    """Gets the relationships asked for
```

Comment:
I would update this to something along the lines of "gets nodes connected via provide relationship"